t/t9210-scalar.sh

```diff
@@ -176,8 +176,9 @@ test_expect_success 'scalar reconfigure' '
 	scalar reconfigure one &&
 	test true = "$(git -C one/src config core.preloadIndex)" &&
 	git -C one/src config core.preloadIndex false &&
-	scalar reconfigure -a &&
-	test true = "$(git -C one/src config core.preloadIndex)"
+	GIT_TRACE2_EVENT="$(pwd)/reconfigure" scalar reconfigure -a &&
+	test true = "$(git -C one/src config core.preloadIndex)" &&
+	test_subcommand git maintenance start <reconfigure
```

Comment:
Do we maybe want to test specifically that the maintenance was re-scheduled? I.e. this instead?

```diff
diff --git a/t/t9210-scalar.sh b/t/t9210-scalar.sh
index 4432a30d10b..83c87ab744e 100755
--- a/t/t9210-scalar.sh
+++ b/t/t9210-scalar.sh
@@ -169,7 +169,9 @@ test_expect_success 'scalar reconfigure' '
 	scalar reconfigure one &&
 	test true = "$(git -C one/src config core.preloadIndex)" &&
 	git -C one/src config core.preloadIndex false &&
+	rm one/src/cron.txt &&
 	scalar reconfigure -a &&
+	test_is_file one/src/cron.txt &&
 	test true = "$(git -C one/src config core.preloadIndex)"
 '
 
```